src/components/play/PlayUI.js

```diff
@@ -47,16 +47,19 @@ class PlayUI extends Component {
     async componentDidMount() {
         // register this component with parent so we can do some instant updates bypassing redux for speed
         this.props.childRef(this)
-
         await this.createRound()
-        window.addEventListener('resize', this.onWindowResizeThrottled)
-        window.addEventListener('keypress', this.onKeypress)
-        window.addEventListener('dblclick', () => this.onMuteToggle(this.props))
+        if (window) {
```

Comment:
This should always be defined

Reply:
I agree but for some reason I got an undefined window error a couple of times on localhost.